src/rules/media-feature-name-case/README.md

```diff
@@ -0,0 +1,73 @@
+# media-feature-name-case
+
+Specify lowercase or uppercase for media feature name.
```

Comment:
This should be plural: `Specify lowercase or uppercase for media feature names.`, i.e. `names.`, not `name.`